_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: default
+title: OTel metrics raw
```

Comment:
```suggestion
title: `otel_metrics_raw_processor`
```

Reply:
We can't use tic marks in the title or in headings. In all other instances, I will apply this change. Thanks!